### PR TITLE
A few TypeScript declaration updates

### PR DESCRIPTION
**Pull request by kompfner**

### CODE REVIEW

**Comment by kompfner** [comment]
@philmillman, discovered these while working on the corresponding RN update

**Review by philmillman — CHANGES_REQUESTED** [review]

File: index.d.ts [hunk 1]
```diff
@@ -449,15 +448,18 @@ export interface DailyCall {
...
   load(properties?: DailyLoadOptions): Promise<void>;
   startScreenShare(captureOptions?: DailyScreenCaptureOptions): void;
   stopScreenShare(): void;
   startRecording(): void;
   stopRecording(): void;
+  startLiveStreaming(options: {
+    rtmpUrl: string;
+    width?: number;
+    height?: number;
+    backgroundColor?: string;
```
[the remark this answers is not in the document]
Comment (kompfner) [reply index.d.ts:460]: huh, looking at the code it does seem like these arguments are passed all the way to the gstreamer script. or do you mean that if we pass in anything other than the defaults (1920 x 1080, #0xff000000) it's untested and might yield unexpected results?

[the remark this answers is not in the document]
Comment (philmillman) [reply index.d.ts:460]: IIRC the settings are fixed at 1080p @ 20fps, keyframe every 3s, 5mbit, @kwindla ??

[the remark this answers is not in the document]
Comment (philmillman) [reply index.d.ts:460]: Or possibly only tested at those settings

[the remark this answers is not in the document]
Comment (kwindla) [reply index.d.ts:460]: @philmillman is right that we've only done significant testing with the default width and height. (I have tested with the background color set to different values.)

My suggestion is that we remove `width`/`height` from the .ts definitions, but leave `backgroundColor`.

**Review by kwindla — APPROVED** [review]
lgtm other than the `width` and `height` arguments to `startLiveStreaming()`.

**Comment by kompfner** [comment]
Merging with `width` and `height` removed

**Pull request merged**